tests/integ/test_dataframe.py

```diff
@@ -2437,10 +2437,22 @@ def test_describe(session):
     assert "invalid identifier" in str(ex_info)
 
 
+
+
+def test_truncate_existing_table(
+    session
+):
```

Comment:
```suggestion
def test_truncate_existing_table(session):
```
nit: would you try running `pre-commit run` for automatically linting files. There are a few lints that can be auto fixed